src/dmap_import/util_rds.py

```diff
@@ -230,6 +257,14 @@ def select_as_list(
         with self.session.begin() as cursor:
             return [row._asdict() for row in cursor.execute(select_query)]
 
+    def vaccuum_analyze(self, table: Any) -> None:
+        """RUN VACUUM (ANALYZE) on table"""
+        table_as = self._get_schema_table(table)
+
+        with self.session.begin() as cursor:
+            cursor.execute(sa.text("END TRANSACTION;"))
+            cursor.execute(sa.text(f"VACUUM (ANALYZE) {table_as};"))
+
```

Comment:
Running `vaccuum (analyze)` after large bulk table changes is recommended, most useful for `device_events` table.